### ISSUE TITLE
High DPI Icon

### ISSUE THREAD

**Issue by afscrome** [microsoft/perfview#313]
The current icon doesn't look great on high dpi screens.

![image](https://user-images.githubusercontent.com/289860/28321417-25d6cd34-6bcb-11e7-8272-b9d4b24a7a57.png)

**Comment by brianrob** [comment]
You are right.  These types of fit and finish issues aren't the priority for PerfView development at this time.  If you are interested, please contribute a PR.

**Comment by sharwell** [comment]
Leaving open for community contributions

**Comment by brianrob** [comment]
This is not something we're going to invest in at this time.
